identity/Cargo.toml

```diff
@@ -21,14 +21,16 @@ identity-iota = { version = "=0.5.0-dev.3", path = "../identity-iota", default-f
 
 [dev-dependencies]
 criterion = { version = "0.3" }
-tokio = { version = "1.5", features = ["full"] }
+tokio = { version = "1.14", features = ["full"] }
```

Comment:
This might need correction.

Reply:
It's a dev-dependency so it doesn't really matter. `identity-account` still uses `1.5` but ideally all the projects should migrate to the latest version of `tokio 1.15`.

Reply:
`cargo update` updated this dependency to this version. The newest version I guess i `1.16`. Should we updated them by hand?

Reply:
I think to be consistent the versions should be updated manually yes. Not sure why `1.14` was needed specifically.